ash/src/vk/enums.rs

```diff
@@ -2242,3 +2242,117 @@ impl QueryResultStatusKHR {
     pub const NOT_READY: Self = Self(0);
     pub const COMPLETE: Self = Self(1);
 }
+impl fmt::Debug for ObjectType {
```

Comment:
If possible I would have snuck these in directly below where the type is defined - we already have an edgecase for `Result` in the generator where we can call `generate_const_debugs`? Seems ugly to pass the `const_values` map down into that though, to be able to extract a specific `Ident`.

Reply:
Really doesn't seem worth complicating the generator to me, especially with that code due to be scrapped en masse soon anyway.

Reply:
Fair enough :)

> soon

I've been telling this a lot to myself as well :sweat_smile: